### ISSUE TITLE
Loading pre-trained network without GPU?

### ISSUE THREAD

**Issue by johnmarktaylor91** [rgeirhos/texture-vs-shape#5]
Hello, I'm trying to follow the instructions [here](https://github.com/rgeirhos/texture-vs-shape/blob/master/models/load_pretrained_models.py)  to load the weights for Resnet trained on the texture versus shape stimuli. However, my computer doesn't have a GPU. I tried to get around this by modifying line 23 by adding map_location='cpu'. However, when I do this, line 24 ("load_state_dict") yields the error message: "Missing key(s) in state_dict:", followed by a huge list of the names of all the layers (e.g., "conv1.weight", "layer3.1.bn2.bias", etc.). This error message goes away if I set "strict=True", but I just wanted to check--will this still properly load the network? Or is there some other modification I could try for this to work on a CPU? Resnet50 itself loads just fine on my CPU, but for some reason adding these pre-trained weights is causing problems.

**Comment by rgeirhos** [comment]
I could imagine that the `torch.nn.DataParallel` wrapping in line 22 is causing the problem when loading on a CPU, since this adds a prefix to the state dict keys (which then is not expected for map_location='cpu'). You could loop over state dict key names and remove the prefix, perhaps this helps with the error message?

On the other hand, if you can already load the model but are not 100% sure whether it works as expected, you might want to test whether you get the reported ImageNet validation accuracy using your loaded model.

**Comment by johnmarktaylor91** [comment]
That fixed it, thanks a ton!
